src/sync_ftp/mod.rs

```diff
@@ -92,6 +95,14 @@ impl FtpStream {
             })
     }
 
+    /// ### active_mode
+    ///
+    /// Enable active mode for data channel
+    pub fn active_mode(mut self) -> Self {
+        self.mode = Mode::Active;
+        self
+    }
+
```

Comment:
The constructor is good.
let's create an additional method to change the current mode

```rust
pub fn set_mode(&mut self, mode: Mode) {
   self.mode = mode;
}

Reply:
Ok, done!